crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py

```diff
@@ -671,6 +669,204 @@ def stats_generate_handler(args: argparse.Namespace):
         )
 
 
+def stats_generate_api_task(
+    timescales: List[str],
+    dashboard: BugoutResource,
+    subscription_by_id: Dict[str, BugoutResource],
+):
+    """
+    Start crawler with generate.
```

Comment:
Can you please split this huge function to parts and with comments descirbe what each part do.

Reply:
Only pull out all staff for current `web3`. Whole functions minly filled result S3 object so not sure about correct split.